lsp-methods.el

```diff
@@ -24,6 +24,12 @@
 (require 'inline)
 (require 'em-glob)
 
+(defcustom lsp-workspace-folders-changed nil
```

Comment:
How about `on-workspace-folders-change`? `workspace-folders-changed`

Reply:
Sure, will update the PR.